tests/test_migration.py

```
@@ -30,6 +30,52 @@ async def test_future_version_rejected_and_v2_accepted(hass: HomeAssistant) -> N
+        domain=DOMAIN,
+        data={
+            CONF_CREATE_BINARY_SENSOR: True,
+            CONF_CREATE_SENSOR: False,
+        },
+        version=2,
+    )
+    entry.add_to_hass(hass)
+    monkeypatch.setattr(
+        hass.config_entries,
+        "async_update_entry",
+        lambda *args, **kwargs: False,
```

Ruff (0.16.0)

[warning] Unused lambda argument: `args`

(ARG005)